products/jhipster.md

```diff
@@ -0,0 +1,72 @@
+---
+title: JHipster
+category: app
+versionCommand: yo jhipster
+permalink: /jhipster
+releasePolicyLink: https://www.jhipster.tech/releases/
+changelogTemplate: "https://github.com/jhipster/generator-jhipster/tag/__LATEST__"
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Supported
+#iconSlug: jhipster , WIP see https://github.com/jhipster/generator-jhipster/issues/20533
+
+auto:
+-   npm: https://www.npmjs.com/package/generator-jhipster
+
+releases:
+-   releaseCycle: "7"
+    eol: false
+    support: true
+    latest: "7.9.3"
+    latestReleaseDate: 2022-09-03
```

Comment:
According to https://www.jhipster.tech/releases/, 7.9.3 was released on 2022-09-02.

Reply:
Fixed.